.github/workflows/cspell-us.yml

```diff
@@ -0,0 +1,22 @@
+name: Cspell en_US check
+on: [push]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+
+      - name: Stage Cspell
+        run: |
+          npm i -g cspell
```

Comment:
Try a single line:

```yes | npx cspell "**"```

But also be sure to match our cspell ignores etc. not sure if we are cspelling "**" aka everything

Reply:
"**" respects the `cspell.json` config file @pavlovcik